.gitignore

```diff
@@ -1 +1,3 @@
-/projectEnv/
\ No newline at end of file
+/projectEnv/
+__pycache__
+*.py[co]
```

Comment:
End of Line missing. Check it at all places

Reply:
Yes End of Line missing is resolved